bigbluebutton-html5/imports/ui/components/waiting-users/styles.scss

```diff
@@ -0,0 +1,143 @@
+@import "/imports/ui/stylesheets/mixins/focus";
+@import "/imports/ui/stylesheets/variables/_all";
+
+.panel {
+  background-color: #fff;
+  padding: var(--md-padding-x);
+  display: flex;
+  flex-grow: 1;
+  flex-direction: column;
+  overflow: hidden;
+  height: 100vh;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  align-items: left;
+  flex-shrink: 0;
+
+  a {
+    @include elementFocus(var(--color-primary));
+    padding-bottom: var(--sm-padding-y);
+    padding-left: var(--sm-padding-y);
+    text-decoration: none;
+    display: block;
+  }
+
+  [class^="icon-bbb-"],
+  [class*=" icon-bbb-"] {
+    font-size: 85%;
+  }
+}
+
+.title {
+  @extend %text-elipsis;
+  flex: 1;
+
+  & > button, button:hover {
+    margin-top: 0;
+    padding-top: 0;
+    border-top: 0;
+  }
+}
+
+.hideBtn {
+  position: relative;
+  background-color: var(--color-white);
+  display: block;
+  margin: 4px;
+  margin-bottom: 2px;
+  padding-left: 0px;
+
+  > i {
+    color: black;
+  }
+
+  &:hover {
+    background-color: var(--color-white);
+  }
+}
+
+.customBtn {
+  width: 100%;
+  padding: .75rem;
+  margin: .3rem 0;
+  font-weight: 400;
+  font-size: var(--font-size-base);
+}
+
+.mainTitle {
+  color: var(--color-gray);
+}
+
+.userAvatar {
+  width: 2rem;
+  margin: .5rem;
+}
+
+.listItem {
+  display: flex;
+  flex-flow: row;
+  flex-direction: row;
+  align-items: center; 
+  justify-content: space-between;
+  transition: all calc(var(--enableAnimation) * .3s);
```

Comment:
Perhaps `   [style~="--enableAnimation:1;"] & {...` like in https://github.com/bigbluebutton/bigbluebutton/pull/6832/files ? so it works in more browsers?